### PR TITLE
sys-block/targetcli-fb: bump to 2.1.53

### PR DESCRIPTION
**Pull request by diogocp**
Bug: https://bugs.gentoo.org/722674
Bug: https://bugs.gentoo.org/736086
Closes: https://bugs.gentoo.org/718528

### CODE REVIEW

File: sys-block/targetcli-fb/targetcli-fb-2.1.53.ebuild [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
```
Comment (thesamesam) [inline sys-block/targetcli-fb/targetcli-fb-2.1.53.ebuild:6]: Python 3.9 too?

Comment (diogocp) [reply sys-block/targetcli-fb/targetcli-fb-2.1.53.ebuild:6]: Not supported by `dev-python/configshell-fb` or `dev-python/rtslib-fb`.

Comment (thesamesam) [reply sys-block/targetcli-fb/targetcli-fb-2.1.53.ebuild:6]: It's fine for now, but can you make sure bugs are filed for those and/or add it yourself if tests pass for those?

Comment (thesamesam) [reply sys-block/targetcli-fb/targetcli-fb-2.1.53.ebuild:6]: ping

**Pull request closed without merging**